src/ignitus-StudentSignUp/signUp/Components/signUp.js

```diff
@@ -0,0 +1,60 @@
+/* eslint-disable react/jsx-filename-extension */
+/* eslint-disable jsx-a11y/click-events-have-key-events */
+/* eslint-disable jsx-a11y/no-static-element-interactions */
+import React, {useState} from 'react';
+import {Link} from 'react-router-dom';
+import progress from '../../../ignitus-Assets/Images/progressPage1.png';
+import imageMap from '../../../ignitus-Assets/Images/profileImageMap.png';
+import {withErrorBoundary} from '../../../ignitus-Internals';
+
+import '../Styles/style.scss';
+
+const signUp = () => {
+  return (
```

Comment:
same here you can omit return like here we didi it like this, https://github.com/Ignitus/Ignitus-client/blob/master/src/ignitus-AboutPage/ignitus-CoreTeam/Components/team.tsx

Reply:
Ok. I will do this.

Reply:
Thanks @gargshruti30 👍